src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -304,14 +304,23 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
     };
 
     updateTaskAssignment = (taskId: string, taskAssignmentId: string, status: TaskCollabStatus): void => {
-        const { file, api } = this.props;
+        const { file, api, onTaskAssignmentUpdate } = this.props;
+        // $FlowFixMe
```

Comment:
Can we add more information about why we're suppressing flow here? It seems risky to override the typing system when it comes to state, since if `currentUser` is undefined, this could throw an error.

Reply:
Can you take a look again ?